samples/build-duration-logger/index.html

```diff
@@ -10,28 +10,63 @@
     const gradleEnterpriseServer = 'https://gradle.my-company.com'; // replace with your Gradle Enterprise base url
     const buildStreamUrl = `${gradleEnterpriseServer}/build-export/v1/builds/since/now?stream`;
     const buildStream = new EventSource(buildStreamUrl);
-    const buildEventStreamUrl = buildId => `${gradleEnterpriseServer}/build-export/v1/build/${buildId}/events?eventTypes=BuildStarted,BuildFinished`;
+
+    class BuildProcessor {
+
+        constructor(gradleEnterpriseServer, maxConcurrentBuildsToProcess) {
+            this.gradleEnterpriseServer = gradleEnterpriseServer;
+            this.buildIds = [];
```

Comment:
I would use `pendingBuildIs` to convey that these are the build ids waiting to be processed.

Reply:
Done